src/App/integrationTests/collectRecords/observersInput.test.js

```diff
@@ -25,7 +25,7 @@ test('Observers input shows users that have been removed from the project and al
   const observersRow = await screen.findByTestId('observers')
 
   expect(observersRow).toHaveTextContent(
-    'Betsy Craig was an observer on this sample unit but is no longer in this project.',
+    'Betsy Craig was an observer on this sample unit but is no longer a part of this project.',
   )
```

Comment:
Well, here is the argument for running the tests in the CI. I forgot to run them locally after I made this tweak on another ticket.